Gems/LmbrCentral/Code/Source/Shape/EditorCapsuleShapeComponent.cpp

```diff
@@ -62,12 +76,26 @@ namespace LmbrCentral
         EditorBaseShapeComponent::Activate();
         m_capsuleShape.Activate(GetEntityId());
         AzFramework::EntityDebugDisplayEventBus::Handler::BusConnect(GetEntityId());
+        const AZ::EntityComponentIdPair entityComponentIdPair(GetEntityId(), GetId());
+        AzToolsFramework::CapsuleManipulatorRequestBus::Handler::BusConnect(entityComponentIdPair);
+        AzToolsFramework::RadiusManipulatorRequestBus::Handler::BusConnect(entityComponentIdPair);
+        AzToolsFramework::ShapeManipulatorRequestBus::Handler::BusConnect(entityComponentIdPair);
 
         GenerateVertices();
+
+        // ComponentMode
```

Comment:
question: Is this comment needed?

Reply:
No, don't think so, another copy-paste thing. There are comments like that all over the place which I'm sure could be removed.